cohorts/functions.py

```diff
@@ -96,7 +96,7 @@ def neoantigen_count(row, cohort, filter_fn=None,
     return np.nan
 
 def expressed_missense_snv_count(row, cohort, **kwargs):
-    filter_fn = kwargs.pop("filter_fn")
+    filter_fn = kwargs.pop("filter_fn", None)
```

Comment:
@arahuja glad you asked, since this was wrong! Felt a little unsure so wrote a test that revealed the problem. Thanks for pointing it out.